tests/php/Controllers/CMSMainTest.php

```diff
@@ -611,7 +611,7 @@ public function testListViewForm(array $params, ?int $limit, int $expectedPreLim
         $page12 = $this->objFromFixture(SiteTree::class, 'page12');
         // Deleted
         $page1->doUnpublish();
-        $page1->delete();
+        $page1->suppressDeletedVersion(fn () => $page1->delete());
```

Comment:
We should probably also add a properly archived page here, which is what I think `CMSSiteTreeFilter_StatusDeletedPages` is meant to show. Lemme know if you want that (I'd do it in `6.0` or `5.4` since it's just testing existing stuff, not directly related to the changes at hand